report_xlsx/models/res_currency.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+
+class ResCurrency(models.Model):
+    _inherit = "res.currency"
+
+    def _get_format_amount(self):
```

Comment:
Too generic. Maybe something like:
```suggestion
    def _report_xlsx_format_amount(self):
```

Anyway shouldn't it be in the abstract?